server/backend/opensearch_security_client.ts

```diff
@@ -220,7 +225,8 @@ export class SecurityClient {
   public async authToken(
     requestId: string | undefined,
     samlResponse: any,
-    acsEndpoint: any | undefined = undefined
+    acsEndpoint: any | undefined = undefined,
+    authRequestType: string | undefined
```

Comment:
Nit: I saw this on other APIs as well, can we avoid to have multiple possible undefined/any parameters and instead replace this with a typed/named object? It might get confusing with how to call something where several of the later options are optional or undefined. Not sure if that should be addressed here or we need another issue for repo cleanup.